fusor-ember-cli/app/controllers/assign-nodes.js

```diff
@@ -246,66 +247,66 @@ export default Ember.Controller.extend({
       });
       var token = Ember.$('meta[name="csrf-token"]').attr('content');
 
-      me.set('loadingSpinnerText', "Saving...");
-      me.set('showLoadingSpinner', true);
-      Ember.$.ajax({
-        url: '/fusor/api/openstack/deployments/' + this.get('deploymentId') + '/deployment_plans/' + plan.get('id') + '/update_parameters',
+      self.set('loadingSpinnerText', "Saving...");
+      self.set('showLoadingSpinner', true);
+      console.log('action: saveRole, PUT /fusor/api/openstack/deployments/' + this.get('deploymentId') + '/deployment_plans/overcloud/update_parameters');
+      //ic-ajax request
+      request({
+        url: '/fusor/api/openstack/deployments/' + this.get('deploymentId') + '/deployment_plans/overcloud/update_parameters',
         type: 'PUT',
         headers: {
           "Accept": "application/json",
           "Content-Type": "application/json",
           "X-CSRF-Token": token,
         },
-        data: JSON.stringify({ 'parameters': params }),
-        success: function() {
+        data: JSON.stringify({ 'parameters': params })
+      }).then( function() {
```

Comment:
ic-ajax library doesn't have the `success:` hook. recommended to to use the js promise `.then()` method instead.